recipe-server/client/control/components/Fields.js

```diff
@@ -50,34 +50,43 @@ buildControlField.propTypes = {
   children: pt.node,
 };
 
+export const CheckboxGroup = ({ name, onChange, options = [], value }) => {
+  /**
+   * Checkbox change event handler. Appends or removes the selected checkbox's
+   * value to the existing `value` prop, and reports the change up to redux-form.
+   *
+   * @param  {Event} onChange event object
+   */
+  const handleChange = ({ event: target }) => {
```

Comment:
If we make a child component (`CheckboxGroupField` maybe?) and add this as a method on it, we can avoid unnecessary re-renders due to the function comparison issue.